Makefile

```diff
@@ -11,10 +11,16 @@ deps:
 
 .PHONY: cover
 cover:
-	go test -coverprofile=coverage.out ./...
+	go test -short -coverprofile=coverage.out ./...
 	go tool cover -html=coverage.out
 	rm coverage.out
 
+.PHONY: test-short
+test-short:
+	go test ./fd/ -v -count 1 -short
+	go test ./pipeline/ -v -count 1 -short
+	go test ./plugin/... -v -count 1 -short
```

Comment:
so we need to manage this list here? what if someone forgets to add here?

Reply:
it's same as `make test` but with `-short` flag.